xamarin/Xamarin.Forms
# [iOS] Shell flyout header on iOS now is dynamic based on the header content

### Description of Change ###
- Made it so the content height gets measured out
- changed minsize from 44 to 56 to match android

### Issues Resolved ### 
- fixes #3875

### Platforms Affected ### 
- iOS

### Testing Procedure ###
I updated store shells height to be a number that's not 200 (which was the previous default) so you can use store shell to verify android and iOS are the same size. Also click on the different FH scroll behaviors

### PR Checklist ###
